### PR TITLE
Case 28271; Update Drupal project template composer.json

### PR DESCRIPTION
**Pull request by kungfuchris**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,28 +19,28 @@
...
+        "drupal/coder": "~2.11.0",
         "drupal/config_update": "1.*@dev",
-        "drupal/devel": "1.*@dev",
-        "drupal/drupal-extension": "~3.0",
+        "drupal/devel": "~1.0.0",
+        "drupal/drupal-extension": "~3.2.0",
         "jcalderonzumba/gastonjs": "^1.1@dev",
         "jcalderonzumba/mink-phantomjs-driver": "~0.3.1",
-        "mikey179/vfsStream": "~1.2",
+        "mikey179/vfsStream": "~1.6.0"",
```
Comment (davidgrayston) [inline composer.json]: @kungfuchris double "" typo

File: composer.json [hunk 2]
```diff
@@ -75,7 +76,7 @@
     },
     "extra": {
         "installer-paths": {
-            "web/core": ["type:drupal-core"],
+            "web/core": ["drupal/core"],
```
Comment (davidgrayston) [inline composer.json]: I can't find the reason why we changed this on subs - other project templates seem to use `"web/core": ["type:drupal-core"]` as we had it, so I think we should stick with this if it works

**Comment by davidgrayston** [comment]
@kungfuchris / @marcelovani wondering if we should simplify the project template and move the dependencies into suites, e.g. `dennis_test_suite` (behat, phpunit, ...), `dennis_dev_suite` (devel, ...), `dennis_cli_suite` (drush, console, ...), so it would look something like:
```
    "require": {
        "dennisdigital/dennis_profile": "~1.0.0",
        "dennisdigital/dennis_test_suite": "~1.0.0",
        "dennisdigital/dennis_dev_suite": "~1.0.0",
        "dennisdigital/dennis_cli_suite": "~1.0.0",
       ...
    },
```
or we could move drush/console/dev dependencies into `dennis_profile`, and just have `dennis_test_suite`, perhaps for another time?

For this pull request we need to update composer.lock

File: composer.json (hunk 2 above)
Comment (kungfuchris) [reply composer.json]: Ok. Will put what was in the template and ignore what was updated in subs

**Comment by kungfuchris** [comment]
We can move the dependencies into suites later on. Separate task, separate tickets.

**Comment by kungfuchris** [comment]
Closing this as we need to revisit the refactoring into suites at a later date. For now we build the project as suggested my Marcelo for Dash, update the composer.json file to not use the dennis profile

**Pull request closed without merging**